### DISCUSSION TITLE
Discussion notifications

### DISCUSSION THREAD

**Discussion by emveepee** [OpenSageTV/Community#9] [GitHub Discussions usage and questions]
I don't want to see to get email notification so I have enabled "Never notify".  Will the @ username  method still work?  Perhaps someone could reply and try it if no one knows.

**Comment by bartvdbraak** [comment]
Hi @emveepee,

If you have enabled the "Never notify" option for email notifications on this repository, it's understandable that you don't want to receive email notifications for new comments or replies. However, the "@ username" mention functionality on GitHub should still work even if you have disabled email notifications.

To verify this behavior, this reply will mention you by username. If you receive a notification on GitHub, it will confirm that the mention functionality is independent of email notifications. Please let me know if you receive the notification, or if you have any further questions.

Best regards,
Bart

**Reply by wayner9** [reply]
Speaking of discussion notifications, it appears that when I get an email from the forum notifying me of an update the email is from the person's real name.  For example, when @jusjoken responded to a thread I got an email that came from his real name.  Not sure that this is an issue, but people should be aware that their real name is visible on this forum.

**Reply by bartvdbraak** [reply]
If you take a look at @jusjoken's profile, you'll see that he has specified his real name in there. That is something you can take private or use an alias for in Github.

**Reply by wayner9** [reply]
Ok, thanks.  It looks like I don't have my real name specified - do you see my real name or my alias?

**Reply by jusjoken** [reply]
I am fine with my name being know as I use GitHub for other things too.

Good to see people checking things out

**Reply by bartvdbraak** [reply]
@wayner9 
> Ok, thanks. It looks like I don't have my real name specified - do you see my real name or my alias?

Your real name is obfuscated and I can only see your GitHub handle, namely `wayner9`.

@jusjoken 
> I am fine with my name being know as I use GitHub for other things too.
> 
> Good to see people checking things out

Same here!

**Reply by emveepee** [reply]
Thanks I didn't get any email for @emveepee  so never might mean never.

**Comment by LoisGNS** [comment]
I don't have my realname in my Github profile. I did add to the bio a mention of my Sage forums name.

**Comment by JustFred51** [comment]
I don't want to receive email notifications for new postings to Discussions, but I **do** want email for new Issues and/or Pull Requests.  Can't find any way to make this happen.  :^(    Maybe my google-fu is lacking.

**Reply by bartvdbraak** [reply]
To configure your email notifications in GitHub, you can follow these steps to control the notifications for Discussions, Issues, and Pull Requests:

## Repository Settings

1. Sign in to your GitHub account and navigate to the repository where you want to adjust the email notifications.
2. On the repository's main page, click on the "Settings" tab located near the top-right corner.
3. In the left sidebar of the settings page, click on "Notifications."

    ```https://github.com/<name>/<repository>/settings/notifications/edit```

4. Scroll down to the "Email notifications" section. Here you will find different options for configuring your email preferences.
5. To disable email notifications for new postings to Discussions, make sure the "Discussions" option is unchecked.
6. To enable email notifications for new Issues and/or Pull Requests, make sure the respective options for "Issues" and "Pull Requests" are checked.
7. Once you have made your desired changes, scroll to the bottom of the page and click on the "Save" button to apply the settings.

## Account Notifications

You can edit your account notifications by visiting:

    https://github.com/settings/notifications

By adjusting these settings, you can control the types of email notifications you receive from GitHub.
